test/upgrade/check-from-any_version-interval-literals.td

```diff
@@ -27,7 +27,7 @@
 02:03:00
 
 > SELECT * FROM interval_hour;
-"5 days 03:00:00"
+"123:00:00"
```

Comment:
Is this OK to change? The new binary format for Intervals is definitely not backwards compatible with previous versions. Do we care about that right now? Do I need to write some type of migration? Or is fine for now because persistence isn't fully in place yet?

Reply:
We'll need version specific tests for this because it won't work w/ any version. You can find the details about what to do here: test/upgrade/README.md but lmk if you want to walk through it